pkg/controller/networkpolicy/networkpolicy_controller.go

```diff
@@ -1146,6 +1183,31 @@ func podToGroupMember(pod *v1.Pod, includeIP bool) *controlplane.GroupMember {
 	return memberPod
 }
 
+func nodeToGroupMember(node *v1.Node) (member *controlplane.GroupMember) {
+	member = &controlplane.GroupMember{Node: node.Name}
+	nodeIPs, err := k8s.GetNodeAddrs(node)
+	if err != nil {
```

Comment:
no reason to not include other IPs when it fails to get some IPs, but we should log the error as they are not expected.

Reply:
Updated.

Reply:
Would be better to change like this?
```
func nodeToGroupMember(node *v1.Node) (member *controlplane.GroupMember) {
	member = &controlplane.GroupMember{Node: &controlplane.NodeReference{Name: node.Name}}
	ips, err := k8s.GetNodeAllAddrs(node)
	if err != nil {
		klog.ErrorS(err, "get Node IP addresses error", "Name", node.Name)
	}
	for ip := range ips {
		member.IPs = append(member.IPs, ipStrToIPAddress(ip))
	}
	return
}
```

Reply:
Yes